code_schemes/icraf_s01e02.json

```diff
@@ -3,69 +3,373 @@
+        "NumericValue": 18,
+        "StringValue": "ubuntu_power_company",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-72e7b87c",
+        "CodeType": "Normal",
+        "DisplayText": "kfs",
+        "NumericValue": 19,
+        "StringValue": "kfs",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-13b34240",
+        "CodeType": "Normal",
+        "DisplayText": "nema",
+        "NumericValue": 20,
+        "StringValue": "nema",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-9c1b004c",
+        "CodeType": "Normal",
+        "DisplayText": "kari",
+        "NumericValue": 21,
+        "StringValue": "kari",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-d2f07841",
+        "CodeType": "Normal",
+        "DisplayText": "self help groups",
+        "NumericValue": 22,
+        "StringValue": "self_help_groups",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-23290d55",
+        "CodeType": "Normal",
+        "DisplayText": "individual champions",
+        "NumericValue": 23,
+        "StringValue": "individual_champions",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-2b25f63f",
+        "CodeType": "Normal",
+        "DisplayText": "kariara water resource users association",
+        "NumericValue": 24,
+        "StringValue": "kariara_water_resource_users_association",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-c913d906",
+        "CodeType": "Normal",
+        "DisplayText": "kamae potato farmers",
+        "NumericValue": 25,
+        "StringValue": "kamae_potato_farmers",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-d4da98dc",
+        "CodeType": "Normal",
+        "DisplayText": "nana fibre and sisal baskets",
+        "NumericValue": 26,
+        "StringValue": "nana_fibre_and_sisal_baskets",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-34c64372",
+        "CodeType": "Normal",
+        "DisplayText": "national agricultural and rural inclusive program",
+        "NumericValue": 27,
+        "StringValue": "national_agricultural_and_rural_inclusive_program",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-ad9818b2",
+        "CodeType": "Normal",
+        "DisplayText": "ifad",
+        "NumericValue": 28,
+        "StringValue": "ifad",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-ff302452",
+        "CodeType": "Normal",
+        "DisplayText": "unesco",
+        "NumericValue": 29,
+        "StringValue": "unesco",
+        "VisibleInCoda": true
+        },
+        {
+        "CodeID": "code-c28fa84a",
+        "CodeType": "Normal",
+        "DisplayText": "ldk(K)ltd athi river",
+        "NumericValue": 30,
+        "StringValue": "ldk(K)ltd_athi_river",
```

Comment:
maybe substitute the brackets for _? Not sure

Reply:
Updated thanks